### ISSUE TITLE
[v5 Request] Sponge Support

### ISSUE THREAD

**Issue by HeroCC** [Hidendra/LWC#854]
Hello!

I read on the v5 Wiki that Forge, Bukkit, etc will be supported, all from the same JAR file. Is there a chanse you can add sponge to this? Sponge it the new major server software, now that bukkit got taken down. It is in dev, and by a bunch of devs, including the WorldEdit team, and the Forge team. Thanks!

http://www.spongepowered.org/

**Comment by durand1w** [comment]
Sorry to post to this ticket, but I know a lot of work was done to create a demo on Sponge.  Will there or is there a 1.8 release for Forge or Sponge?  Just trying to plan our upgrade.

**Comment by Time6628** [comment]
LWC Forge will work with sponge. @durand1w

**Comment by durand1w** [comment]
Right. I'm specifically asking if there is, or are plans for, a 1.8 LWC Forge.  We are not expecting to take Sponge at this time, but we might if that is our only option to keep LWC.

**Comment by Time6628** [comment]
@durand1w LWC hasn't been updated in quite some time, so I wouldn't expect it to. What you could do is run a Sponge server with Pore that allows you to run Bukkit plugins on a sponge server.

**Comment by durand1w** [comment]
@Time6628 Thanks for the information.  I'm not sure where the rest of the community will land for 1.8.  I know for our server, we have 22 mods updated for Forge 1.8, some of which I wrote myself to replace Bukkit mods.  We have 12 left, not including Forge LWC.  Additionally, we are expecting to lose functionallity from 6-9 bukkit mods.

Pore as of July 3rd, stated:

"Pore is currently not in a very usable state, as it's still under very heavy development. A large portion of the Bukkit API is still unimplemented, and additionally, at this stage our focus is somewhat shifted away from finding and squashing bugs"

Sponge and Pore might become very popular in the future, but I think for now we are looking towards updating rather than freezing the world on 1.7.10.  Waiting for MCP to update, then Forge, then Sponge, then Pore and then any Bukkit mods will definitely make for long updates.  Hopefully, there is some information on LWC Forge 1.8 since I think LWC is the easiest system to use for locking inventories.  The existing version may already work since it doesn't do anything with blockstates or entities.  Just hoping for some official information on future development plans.
